cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -122,6 +122,9 @@ Instead, the CDI uses the default storage class to allocate scratch space.
 $ oc patch storageclass <STORAGE_CLASS_NAME> -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
 ----
 
+* The {CNVProductName} {CNVVersion} web console includes *slirp* as an option when adding a NIC to a virtual machine, however slirp is not a valid NIC type. Do not select *slirp* when adding a NIC to a virtual machine.
```

Comment:
nits:

s/however/but
s/slirp/*slirp* (in "slirp is not a valid NIC type")
s/when adding/when you add